devnet-sdk/shell/cmd/ctrl/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/controller/surface"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/shell/env"
+	"github.com/urfave/cli/v2"
+)
+
+func run(ctx *cli.Context) error {
+	devnetURL := ctx.String("devnet")
+	action := ctx.String("action")
+	service := ctx.String("service")
+
+	devnetEnv, err := env.LoadDevnetFromURL(devnetURL)
+	if err != nil {
+		return err
+	}
+
+	ctrl, err := devnetEnv.Control()
+	if err != nil {
+		return err
+	}
+
+	lc, ok := ctrl.(surface.ServiceLifecycleSurface)
+	if !ok {
+		return fmt.Errorf("control surface does not support lifecycle management")
+	}
```

Comment:
I would assume that people would like to do this in their go tests - shall we add a tiny helper that encapsulates this block so that they can just do smth like

```go
lc, err := devnetEnv().LifecycleSurface()
```

Reply:
the test-level integration is done by @pcw109550 in a separate PR
This command here is just for illustration purpose.